tests/app/test_internal_user_service.py

```diff
@@ -28,6 +45,26 @@ def test_results_default_information_returned_for_group_user(self):
 
         self.assertEqual(expected, actual)
 
+    @requests_mock.mock()
+    def test_default_user_returned_if_not_http_error_404(self, mock_request):
+        HTTP_ERROR_USER = "bb7c51c0-96b7-441d-9881-4ad1a3d3d396"
+        with self.app.app_context():
+            with current_app.test_request_context():
+                uaa_url = f"{current_app.config['UAA_URL']}/Users/{HTTP_ERROR_USER}"
+                mock_request.get(uaa_url, status_code=401, reason="Forbidden", text="{}")
+                with self.assertRaises(HTTPError):
+                    InternalUserService().get_user_details(HTTP_ERROR_USER)
+
+    @requests_mock.mock()
+    def test_default_user_returned_if_http_error_404(self, mock_request):
+        DELETED_USER = "bb7c51c0-96b7-441d-9881-4ad1a3d3d396"
```

Comment:
As above with Constant

Reply:
Isn't that also the same UUID?

Reply:
Deleted doesn't mean much, I would use missing or something similar

Reply:
The same UUID is being used. I've extracted it out like above